Extensions/xkit_patches.js

```diff
@@ -236,6 +282,17 @@ XKit.tools.getParameterByName = function(name){
 	}
 };
 
+/**
+ * @return {Object} An overview of the browser's information:
+ *	name<"Google Chrome" | "Mozilla Firefox" | "Apple Safari">: The browser's human-readable name
```

Comment:
The string part is from Typescript's String Literal Types specification as described in https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#string-literal-types.

The description of keys using parameterization is something I stopped doing later and I will update this location to the one I eventually decided on which was "name: Type - description"

What is your preference?